### ISSUE TITLE
[Violation] 'click' handler took 1000ms

### ISSUE THREAD

**Issue by suraj740** [ngx-formly/ngx-formly#2453]
I have large model data of around 200KB, i using repeat section to add repeated fields, but after clicking on `Add` button it takes more than 1000ms of time to response on that click event.

I think this should not happen irrespective of data size.

Attached screenshot for the same.

<img width="256" alt="Screenshot 2020-08-19 at 1 36 42 AM" src="https://user-images.githubusercontent.com/24958883/90560290-7fa76700-e1bc-11ea-9191-4f6a83093c68.png">

Thanks!!

**Comment by aitboudad** [comment]
Hi, could you please try the dev version `5.10.0-dev.ad26817d` and enable `lazyRender`:

```patch
    FormlyModule.forRoot({
+     extras: {
+       lazyRender: true
+     },
    }),
```

**Note**: Formly could be only suitable if you want to handle small dataset as It may have an impact on performance, I would prefer to use a `datatable` with an edit button for each row to open the form on demand using dialog or a popover.

**Comment by suraj740** [comment]
@aitboudad  Thanks , i have tried installing the dev version:
```
"@ngx-formly/core": "^5.10.0-dev.ad26817d",
"@ngx-formly/material": "^5.0.0",
"@ngx-formly/schematics": "^5.10.0-dev.ad26817d"
```

But when i add
```
 FormlyModule.forRoot({
    extras: {
       lazyRender: true
    }
 })
```
It gives me error as `Type '{ lazyRender: true; }' is not assignable to type '{ fieldTransform?: any; immutable?: boolean; showError?: (field: FieldType<FormlyFieldConfig>) => boolean; checkExpressionOn?: "modelChange" | "changeDetectionCheck"; }'.`

and still it has same behaviour as it was earlier.

Thanks.

**Comment by aitboudad** [comment]
all Formly packages should match the same version:

```patch
"@ngx-formly/core": "^5.10.0-dev.ad26817d",
- "@ngx-formly/material": "^5.0.0",
+ "@ngx-formly/material": "^5.10.0-dev.ad26817d",
"@ngx-formly/schematics": "^5.10.0-dev.ad26817d"
```

**Comment by suraj740** [comment]
Ok i did, now error gone.
But still i am facing the load time issue after button click.
I have button on some fileds, when i click it takes more time to open that in new tab.

**Comment by aitboudad** [comment]
Without a reproduction, I can't help but my guess is due to the number of components that you're willing to render at once.
You may check some hints here https://github.com/ngx-formly/ngx-formly/issues/1489#issuecomment-479651796.

Feel free to reopen if you've got a reproduction example using out `stackblitz` starter example!

**Comment by aitboudad** [comment]
**Note**: `lazyRender` is now available in [v5.10.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.0) release.

**Comment by suraj740** [comment]
Thanks will check it out.
